client/packages/system/src/Patient/DetailView/DetailView.tsx

```diff
@@ -0,0 +1,23 @@
+import React, { FC } from 'react';
+import {
+  DetailViewSkeleton,
+  useJsonForms,
+  useParams,
+} from '@openmsupply-client/common';
+
+export const PatientDetailView: FC = () => {
+  const { patientId } = useParams();
+  const { JsonForm, loading, error } = useJsonForms(patientId);
+
+  if (loading) return <DetailViewSkeleton hasGroupBy={true} hasHold={true} />;
+
+  return (
+    <React.Suspense
+      fallback={<DetailViewSkeleton hasGroupBy={true} hasHold={true} />}
```

Comment:
maybe we should make an issue for a 'form skeleton'?

Reply:
Yeah, I couldn't really see how it looked here cos I'm using hard-coded data. When I see how bad this one is (or isn't?) when I use the database, I'll make an issue then.